### PR TITLE
Feature: Locations

### PR DESCRIPTION
**Pull request by duytantran**

### CODE REVIEW

**Review by duytantran — COMMENTED** [review]
LGTM

File: csharp/src/Vista.SDK/GmodNode.cs [hunk 1]
```diff
@@ -36,6 +38,34 @@ internal GmodNode(GmodNodeDto dto)
...
+        };
+    }
+
+    public GmodNode TryWithLocation(string? location)
+    {
+        if (string.IsNullOrWhiteSpace(location))
+            return this;
+
+        var locations = VIS.Instance.GetLocations(VisVersion);
+        return this with { Location = locations.TryCreateLocation(location) };
```
Comment (martinothamar) [inline csharp/src/Vista.SDK/GmodNode.cs:57]: If `TryCreateLocation` fails, `Location` will be set to `null`, not sure if thats right

**Comment by duytantran** [comment]
Changed one line in the json-file: 
image

**Review by kristoffersaastad — COMMENTED** [review]
Ser bra ut!! Var bare det vi snakket om jeg har tilbakemeldinger på

**Pull request merged**